aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -465,8 +465,8 @@ define(["dojo/_base/declare",
          }
          else
          {
-            var publishGlobal = this.publishGlobal !== false;
-            var publishToParent = this.publishToParent !== false;
+            var publishGlobal = this.publishGlobal || false;
+            var publishToParent = this.publishToParent || false;
             this.alfPublish(this.publishTopic, this.publishPayload, publishGlobal, publishToParent);
```

Comment:
Could all be written on one line, if desired, including forcing to boolean (is that really required??) ...

```
this.alfPublish(this.publishTopic, this.publishPayload, !!this.publishGlobal, !!this.publishToParent);
```